src/routes/tools/annual-averages/index.svelte

```diff
@@ -67,10 +46,13 @@
   import { onMount } from "svelte";
   import { Loading } from "carbon-components-svelte";
   import { inview } from "svelte-inview/dist/";
+  import { stores } from "@sapper/app";
```

Comment:
Because we're typically importing so many stores, I might alias this as `sapperStores` so that it's a little more clear in the code what it's referring to. e.g.

```js
import { stores as sapperStores } from "@sapper/app";
```

Reply:
good idea. fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/179/commits/11a328605b27810fc05273d28ed06df7fa1a6e40